standard/documentation-comments.md

```diff
@@ -304,16 +309,19 @@ where
 **Example:**
 
 ```csharp
-/// <summary>This is the entry point of the Point class testing program.
-/// <para>
-/// This program tests each method and operator, and
-/// is intended to be run after any non-trivial maintenance has
-/// been performed on the Point class.
-/// </para>
-/// </summary>
-public static void Main() 
+public class Point
```

Comment:
Maybe rename this to PointTest? (It's likely not to be in Point itself.)

Reply:
Done